pom.xml

```diff
@@ -204,8 +203,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven-compiler-plugin.version}</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
> Saying this, no clue if 2.12.x supports JDK7?

2.12 targets JDK8 and includes compiler changes that generate JDK7-incompatible objects like lambdas and interfaces with default methods. So indeed, moving to Scala 2.12 means dropping JDK7 support.